plugins/heketi/transactions.go

```diff
@@ -0,0 +1,37 @@
+package heketi
+
+import (
+	"fmt"
+	"os/exec"
+
+	heketi "github.com/gluster/glusterd2/plugins/heketi/api"
+	"github.com/gluster/glusterd2/transaction"
+)
+
+func txnHeketiPrepareDevice(c transaction.TxnCtx) error {
+	var deviceinfo heketi.DeviceInfo
+
+	if err := c.Get("nodeid", &deviceinfo.NodeID); err != nil {
+		return err
+	}
+	if err := c.Get("devicename", &deviceinfo.DeviceName); err != nil {
+		return err
+	}
+
+
+	cmd1 := exec.Command("pvcreate", "--metadatasize=128M", "--dataalignment=256K", "/dev/"+deviceinfo.DeviceName)
```

Comment:
Is pvcreate and vgcreate enough for making the device ready for brick usage?
We should create LV post this and using mkfs.xfs create a proper filesyatem. Finally mount the LV in some path to be used a bricks for the volumes.